Tests/MongoSwiftTests/CodecTests.swift

```diff
@@ -649,19 +656,21 @@ final class CodecTests: MongoSwiftTestCase {
         // int32
         let int32 = Int32(5)
 
-        expect(try decoder.decode(AnyBSONValue.self, from: "{ \"$numberInt\" : \"5\" }").value).to(bsonEqual(5))
+        expect(try decoder.decode(AnyBSONValue.self, from: "{ \"$numberInt\" : \"5\" }").value).to(bsonEqual(int32))
 
         let wrappedInt32: Document = ["x": int32]
         expect(try encoder.encode(AnyBSONStruct(int32))).to(equal(wrappedInt32))
         // as int because we convert Int32 -> Int when decoding
```

Comment:
this comment seems out of date.... I don't understand what it's referring to

Reply:
removed